ovos_tts_plugin_server/__init__.py

```diff
@@ -28,8 +28,18 @@ def __init__(self, *args, **kwargs):
 
     @property
     def host(self) -> Optional[str]:
+        self.log.warning("Deprecation Warning: use self.urls instead")
+        return self.urls
+
+    @property
+    def urls(self) -> List[Any]:
         """If using a custom server, set the host here, otherwise it defaults to public servers."""
-        return self.config.get("host", None)
+        urls = self.config.get("host", self.config.get("hosts"))
+        if urls and not isinstance(urls, list):
+            urls = [urls]
+        else:
+            urls = random.shuffle(self.public_servers)
```

Comment:
Since `self.urls` does a new shuffle every time it's referenced, and on new line 69 we call self.urls each time we get audio data, it's still going to shuffle every time.

Reply:
Isn't that what we want? Shuffle the public servers every call? And leave the custom URL's alone?

Reply:
Got it.  It should only shuffle on execution, not on being called.  Ill work more on this today.